.github/workflows/follow-merge-lsf-sync-schedule.yml

```diff
@@ -1,7 +1,8 @@
-name: 'Follow Merge: LSF Sync'
+name: 'Scheduled Label Studio Frontend sync'
```

Comment:
_question_: Good to know!  Is this a modification from the original HumanSignal's github action?

Reply:
Yes it is - I renamed it to be a little more helpful.